examples/seismic/tutorials/14_creating_synthetics.ipynb

```diff
@@ -48,7 +48,8 @@
      "name": "stdout",
```

Comment:
<p>missing plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>Yeah, not sure why that's not showing up. It displays locally. Might be some interplay between the new vtk notebook backend and nbreview?</p>

Reply:
Looks like it doesn't display on github's notebook reader either, but it gets rid of the warning locally